.github/workflows/ci.yml

```diff
@@ -36,6 +36,14 @@ jobs:
             gemfile: rails5.0
           - ruby-version: 2.5
             gemfile: rails5.1
+          - ruby-version: 3.0
+            gemfile: rails6.1
+          - ruby-version: 3.0
+            gemfile: rails7.0
+          - ruby-version: 3.1
+            gemfile: rails6.1
+          - ruby-version: 3.1
+            gemfile: rails7.0
```

Comment:
It feels like testing with 3.0 and 3.1 should be the default. I would add them to the `ruby-version` array in lines 25–26, and then add an `exclude` section for the incompatible Ruby+Rails combinations.

Also, `3.0` should be quoted to avoid it being interpreted as `3`, which installs Ruby 3.1. See https://github.com/actions/runner/issues/849